Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -50,7 +50,7 @@ protected function configureRoute(Route $route, \ReflectionClass $class, \Reflec
         // requirements (@Method)
         foreach ($this->reader->getMethodAnnotations($method) as $configuration) {
             if ($configuration instanceof Method) {
-                $route->setMethods(implode('|', $configuration->getMethods()));
+                $route->setMethods(explode('|', implode('|', $configuration->getMethods())));
```

Comment:
it's ugly, i know.. hehhe
OK, `$configuration->getMethods()` may return 
```
[
     "GET",
     "PUT|POST",
]
```

```
implode('|', $methods); \\ => "GET|PUT|POST"
``` 
as `$route-> setMethods()`works with array, I call explode to normalize